pkgs/by-name/be/bevy_cli/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+
+  pkg-config,
+  openssl,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "bevy_cli";
+  version = "v0.1.0";
```

Comment:
this should also just be "0.1.0"

Reply:
Thanks for catching that. It has been a while 😅 Sorry this was avoidable 🙈